### DISCUSSION TITLE
Does the nuget.config spec allow us to specify Symbol file (.pdb) locations?

### DISCUSSION THREAD

**Discussion by PureKrome** [NuGet/Home#12524] [Q&A]
I love how the nuget.config has `packageSources` and `packageSourceMapping`. That _really_ simplifies people getting started with my projects without having to do custom setups in their GUI of choice.

But .. what about symbol file locations? Sure, Visual Studio (not sure about Rider, etc) auto checks nuget.org for symbols if the nuget package has snupkg's up on nuget.

But it all breaks down terribly if we are using _private_ nuget repositories to store out nuget packages.
We now have to _manually_ add in the Symbol Server to our GUI, like this:

![](https://docs.myget.org/docs/reference/Images/debug-options-2015.png)

So ... would be _really awesome_ if we could have the ability to define where the symbol packages _also exist_ . We already tell the program where our private nuget repo is. Why not where the private symbol server, is?

**Comment by MikeFarrington** [comment]
I totally agree. It would be nice to have this at the solution level (stored in the solution's nuget.config) rather than defined purely in Visual Studio. It'd certainly be one less step to perform when onboarding a new team member. Perhaps this should be suggested via Visual Studio feedback rather than here on the NuGet github Q&A for more attention.
